openapi3/refs.go

```diff
@@ -79,7 +83,8 @@ func (value HeaderRef) GetRef() string { return value.Ref }
 func (value *HeaderRef) ClearRef()     { value.Ref = "" }
 
 func (value *HeaderRef) MarshalJSON() ([]byte, error) {
-	return jsoninfo.MarshalRef(value.Ref, value.Value)
+	return marshalRef(value.Value, value.Ref)
+	//return jsoninfo.MarshalRef(value.Ref, value.Value)
```

Comment:
comment?

Reply:
oops, i unified all of these to call a local function to help address the JSONising, but have forgotten this comment somehow